---
 configure.in |    6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

Index: SDL-1.2.14/configure.in
===================================================================
--- SDL-1.2.14.orig/configure.in	2009-11-27 15:02:19.000000000 -0500
+++ SDL-1.2.14/configure.in	2009-11-27 15:02:24.000000000 -0500
@@ -745,16 +745,12 @@
             esac
         fi
 
-        AC_PATH_PROG(NASM, yasm)
+        AC_PATH_PROG(NASM, nasm)
         echo "%ifidn __OUTPUT_FORMAT__,elf" > unquoted-sections
         echo "section .note.GNU-stack noalloc noexec nowrite progbits" >> unquoted-sections
         echo "%endif" >> unquoted-sections
         CompileNASM unquoted-sections || NASM=""
 
-        if test "x$NASM" = x -o "x$NASM" = x'"$NASM"'; then
-            $as_unset ac_cv_path_NASM
-            AC_PATH_PROG(NASM, nasm)
-        fi
         if test "x$NASM" != x -a "x$NASM" != x'"$NASM"'; then
             AC_DEFINE(SDL_HERMES_BLITTERS)
             SOURCES="$SOURCES $srcdir/src/hermes/*.asm"
