---
 build-scripts/makedep.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/build-scripts/makedep.sh
+++ b/build-scripts/makedep.sh
@@ -69,7 +69,7 @@
         ;;
         asm) cat >>${output}.new <<__EOF__
 
-	\$(LIBTOOL) --tag=CC --mode=compile \$(auxdir)/strip_fPIC.sh \$(NASM) $src -o \$@
+	\$(LIBTOOL) --tag=CC --mode=compile \$(auxdir)/strip_fPIC.sh \$(NASM) -I\$(srcdir)/src/hermes/ $src -o \$@
 
 __EOF__
         ;;
